lib/index.js

```diff
@@ -109,19 +106,16 @@ function sendRequest () {
 
 	deferred.promise.then(function () {
 		// send multi-part request
-		restler.post(argv.host + 'api/apps/' + argv.project + '/coverage?apiKey=' + data.apiKey, {
-			multipart: true,
-			data: data
-		})
-
-		// successful upload
-		.on('success', function (data) {
-			console.log('Comforter Coverage: ' + chalk.green(data.coverage));
-		})
-
-		// network failure or other issue with the request
-		.on('error', fail.bind(null, errors.network))
-		.on('fail', fail.bind(null, errors.badRequest));
+		var config = {
+			url: argv.host + 'api/apps/' + argv.project + '/coverage?apiKey=' + data.apiKey,
+			formData: data
+		};
+
+		request.post(config, function (err, response, body) {
+			if (err) return fail.bind(null, errors.network);
+			if (response.statusCode === 400) return fail.bind(null, errors.badRequest);
```

Comment:
What about other non 2xx responses?

Reply:
According to restler's docs

* `success` is emitted when the request was successful
* `fail` is emitted when the request was successful, but 4XX or 5XX status code returned
* `error` is emitted when some errors have occurred (eg. connection aborted, parse, encoding, decoding failed or some other unhandled errors)

If that's accurate, then comforter-cli currently has no case for other non 2xx responses, so I didn't have anything to map them to with this PR.

Would you like me to add a case? If so, we probably need a new entry in the `errors` object.